database/seed.js

```diff
@@ -1,80 +1,99 @@
-            }
-
-            const products = new Product(randomCategory, randomCompany, randomDescription, randomPrice, randomPrice, randomRating, randomTitle, randomBestSeller)
-
-            let unit = new mainProduct({
-                category: `${products.randomCategory}`,
-                company: `${products.randomCompany}`,
-                description: `${products.randomDescription}`,
-                price: `${products.randomPrice}`,
-                prime: `${products.randomPrime}`,
-                ratings: `${products.randomRating}`,
-                title: `${products.randomTitle}`,
-                best_seller: `${products.randomBestSeller}`
-            }).save(function (err) {
-                if(err) {
-                    console.log(err, 'err from .save')
-                    reject
-                } else {
-                    resolve();
-                }
-            })
-        }
 
-    });
+            // const products = new Product(randomCategory, randomCompany, randomDescription, randomPrice, randomPrice, randomRating, randomTitle, randomBestSeller)
+
+            let randomCategory = faker.commerce.department(); // Clothing
+            let randomCompany = faker.company.companyName(); // Apple Inc.
+            let randomDescription = faker.image.abstract(); // image description
+            let randomPrice = faker.commerce.price(); // $1.00
+            let randomPrime = faker.random.boolean(); // true
+            let randomRating = faker.random.number(); // 4
+            let randomTitle = faker.commerce.productName(); // Swimming Pool
+            let randomBestSeller = faker.random.boolean(); // false
+
+            let mainproduct = new mainProduct({
+                category: `${randomCategory}`,
+                company: `${randomCompany}`,
+                description: `${randomDescription}`,
+                price: `${randomPrice}`,
+                prime: `${randomPrime}`,
+                ratings: `${randomRating}`,
+                title: `${randomTitle}`,
+                best_seller: `${randomBestSeller}`
+            }).save();
+            promisearr.push(mainproduct)
+        }
+        return Promise.all(promisearr).catch(err => console.log(err))
 };
 
 
+// seed photos table
+
+let seedPhotos = function() {
+    var promisearr = [];
 
+        for (var j = 0; j < 100; j++) {
+            let randomPhotoUrl = faker.image.imageUrl(); // https://aws.photo.com
+        
+            let photostable = new photos({
+                photo_url: `${randomPhotoUrl}`
+            }).save()
+            promisearr.push(photostable)
+        }
+    return Promise.all(promisearr).catch(err => console.log(err))
+};
+
+mainProduct.db.dropDatabase(function (err, results) {
+    if (err) {
+    } else {
+      console.log(results);
+    }
+  });
+
+console.log(seedMainProduct(), 'line 60')
 
 const insertSampleProducts = function() {
   mainProduct.create(seedMainProduct())
+    // .then(() => db.close())
```

Comment:
You're already creating your products in seedMainProduct when you run new mainProduct.save(), I don't think they need to be created again. Also since seedMainProducts is async, mainProduct.create is just being passed an unresolved promise at runtime so it will error out.